### PR TITLE
Adding the max_wallclock_seconds setting

### PR DESCRIPTION
**Pull request by mikibonacci**
Addressing https://github.com/aiidalab/aiidalab-qe/issues/709

I have to fix the layout, but in the meanwhile useful to see if there is some major issue.

### CODE REVIEW

**Comment by superstar54** [comment]
@mikibonacci please fix the failed tests.

**Comment by AndresOrtegaGuerrero** [comment]
@mikibonacci i think you are also trying to upload the export folder

**Comment by AndresOrtegaGuerrero** [comment]
@mikibonacci is it possible to add a test for this option?

**Comment by mikibonacci** [comment]
Sure, I will add a test. 
About the export folder, I will check what is happening, I am not sure

File: src/aiidalab_qe/common/widgets.py [hunk 1]
```diff
@@ -762,8 +766,22 @@ def __init__(self, **kwargs):
             description="cpus-per-task",
             style={"description_width": "100px"},
         )
+        self.max_wallclock_seconds = ipw.BoundedIntText(
+            value=3600 * 12,
+            step=3600,
+            min=1800,
+            max=3600 * 24 * 10,
```
[the remark this answers is not in the document]
Comment (AndresOrtegaGuerrero) [reply src/aiidalab_qe/common/widgets.py:773]: The logic should somehow consider the restrictions of the computer (it they are present) , or there should be a help text saying the user to set the walltime depending on the restrictions of the computer you have selected

[the remark this answers is not in the document]
Comment (mikibonacci) [reply src/aiidalab_qe/common/widgets.py]: I added a sentence, and set as max time 24h (10 days was indeed a bit too much).

**Comment by AndresOrtegaGuerrero** [comment]
> @AndresOrtegaGuerrero, I am not sure we need a test for this setting? I adapted the `test_qeapp_computational_resources_widget` function to include also the test for the time. Do you have suggestion on specific tests?

Maybe a for the builder , just to confirm that the builder is indeed generating the walltime change

**Comment by mikibonacci** [comment]
@AndresOrtegaGuerrero my last commit should be the one to put also the check for the builder. 
If it is enough for you, I would like to merge

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM!

**Pull request merged**